codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectEncoder.java

```diff
@@ -47,7 +47,7 @@
  * implement all abstract methods properly.
  */
 public abstract class HttpObjectEncoder<H extends HttpMessage> extends MessageToMessageEncoder<Object> {
-    private static final byte[] CRLF = { CR, LF };
+    protected static final byte[] CRLF = { CR, LF };
```

Comment:
@doom369 this was private by "design" as now a user could get a reference to it and alter the shared byte[]. That said if you make it package private it should be ok.